### PR TITLE
New example using WebGL

### PR DESCRIPTION
**Pull request by Datavizcowboy**

### CODE REVIEW

**Comment by Datavizcowboy** [comment]
webgl_play

**Comment by romsson** [comment]
It seems you can remove some code related to interaction

File: example/webGL/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+//				circleShape.quadraticCurveTo( -circleRadius, circleRadius, 0, circleRadius );
+
+				var extrudeSettings = { amount: 8, bevelEnabled: true, bevelSegments: 2, steps: 2, bevelSize: 1, bevelThickness: 1 };
+
+//				 addShape( shape, color, x, y, z, rx, ry,rz, s );
+
+                var numObjects = 200;
+                    for (var ii = 0; ii < numObjects; ++ii) {
+//                        addShape( triangleShape, extrudeSettings, 0x8080f0, -180, 0, 0, 0, 0, 0, 1 );
+                        addShape( squareShape, extrudeSettings, 0x000000, -300+ii*30, 20, 0, 0, 0, 0, 1 );
```
Comment (romsson) [inline example/webGL/index.html]: Use d3.gridding here instead of iterating over numObjects, and draw the actual x, y, h, w generated by the lib

**Comment by Datavizcowboy** [comment]
Cleaner code and d3-gridding embedded

**Comment by Datavizcowboy** [comment]
webgl_simple

**Comment by romsson** [comment]
Missing libs:

three.js Failed to load resource: the server responded with a status of 404 (Not Found)
stats.min.js Failed to load resource: the server responded with a status of 404 (Not Found)

**Comment by Datavizcowboy** [comment]
webgl_skewed

There is a remaining offset I have to fix

**Comment by Datavizcowboy** [comment]
webgl_draw

**Comment by Datavizcowboy** [comment]
webgl_draw

**Pull request merged**